ccmlib/scylla_node.py

```diff
@@ -710,12 +710,16 @@ def import_bin_files(self):
                 if len(res) > 1:
                     raise RuntimeError('{}: found too make matches: {}'.format(search_pattern, res))
                 loader = res[0]
-                cmd = ['patchelf', '--set-interpreter', loader, dst]
-                subprocess.check_call(cmd)
 
                 self._launch_env = dict(os.environ)
                 self._launch_env['LD_LIBRARY_PATH'] = dbuild_so_dir
 
+                cmd = [loader, os.path.join(dbuild_so_dir, 'patchelf'), '--set-interpreter', loader, dst]
+                p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=self._launch_env)
+                (stdout, stderr) = p.communicate()
```

Comment:
Why aren't you using subprocess.check_call?

Reply:
Because it doesn't support `env=` (needed for setting `LD_LIBRARY_PATH`, needed for running `patchelf` from the the reloc dir)

Reply:
ok

Reply:
btw, it's not documented very good, but you can pass env to `subprocess.check_call()`:
```
Python 2.7.15rc1 (default, Nov 12 2018, 14:31:15) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import subprocess
>>> subprocess.check_call('env', env={'fruch': 'test'})
fruch=test
0
>>> 
```